### PR TITLE
Support pay as you go scalardl in aws marketplace

### PR DESCRIPTION
**Pull request by kota2and3kan**
**Note: We have to merge this PR after we publish pay-as-you-go ScalarDL on AWS Marketplace. We are working on it now.**

This PR adds the `how to deploy pay-as-you-go ScalarDL` section in the AWS Marketplace guide.

Also, I fixed product names in this document (`Scalar DB` -> `ScalarDB` / `Scalar DL` -> `ScalarDL`).

Please take a look!

### CODE REVIEW

File: docs/AwsMarketplaceGuide.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # How to install Scalar products through AWS Marketplace
 
-Scalar products (Scalar DB, Scalar DL, and their tools) are provided in AWS Marketplace as Container Image. This guide explains how to install Scalar products through AWS Marketplace.
+Scalar products (ScalarDB, ScalarDL, and their tools) are provided in AWS Marketplace as Container Image. This guide explains how to install Scalar products through AWS Marketplace.
```
Comment (josh-wong) [inline docs/AwsMarketplaceGuide.md:3]:
```suggestion
Scalar products (ScalarDB, ScalarDL, and their tools) are available in the AWS Marketplace as container images. This guide explains how to install Scalar products through the AWS Marketplace.
```

File: docs/AwsMarketplaceGuide.md [hunk 2]
```diff
@@ -20,14 +22,102 @@ Note that some Scalar products are licensed under commercial licenses, and the A
...
+       --region <REGION> \
+       --cluster <EKS_CLUSTER_NAME> \
+       --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringFullAccess \
+       --approve \
+       --override-existing-serviceaccounts
+   ```
+
+1. Update the custom values file of the Helm Chart of a Scalar product you want to install.
+
+   You need to specify the private container registry (ECR) of AWS Marketplace and the version (tag) as the value of `[].image.repository` and `[].image.version (tag)` in the custom values file. You also need to specify the service account name that you created in the previous step as the value of `[].serviceAccount.serviceAccountName` and set `true` to `[].serviceAccount.automountServiceAccountToken`.
```
Comment (josh-wong) [inline docs/AwsMarketplaceGuide.md:56]:
```suggestion
   You need to specify the private container registry (ECR) of the AWS Marketplace and the version (tag) as the value for `[].image.repository` and `[].image.version (tag)` in the custom values file. You also need to specify the service account name that you created in the previous step as the value for `[].serviceAccount.serviceAccountName` and set `[].serviceAccount.automountServiceAccountToken` to `true`.
```

File: docs/AwsMarketplaceGuide.md [hunk 3]
```diff
@@ -20,14 +22,102 @@ Note that some Scalar products are licensed under commercial licenses, and the A
...
+        ```
+      * ScalarDL Schema Loader for Auditor (schema-loader-auditor-custom-values.yaml)
+        ```yaml
+        schemaLoading:
+          image:
+            repository: "709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalardl-schema-loader-auditor-aws-payg"
+            version: "3.8.0"
+        ```
+
+1. Deploy the Scalar products using the Helm Chart with the above custom values files.
```
Comment (josh-wong) [inline docs/AwsMarketplaceGuide.md:94]:
```suggestion
1. Deploy Scalar products by using Helm Charts in conjunction with the above custom values files.
```

File: docs/AwsMarketplaceGuide.md [hunk 4]
```diff
@@ -20,14 +22,102 @@ Note that some Scalar products are licensed under commercial licenses, and the A
...
+        ```
+      * ScalarDL Schema Loader (Auditor)
+        ```console
+        helm install schema-loader scalar-labs/schema-loading -f ./schema-loader-auditor-custom-values.yaml
+        ```
+
 ## **[BYOL]** Deploy containers on EKS (Amazon Elastic Kubernetes Service) from AWS Marketplace using Scalar Helm Charts
 
-If you subscribe to Scalar products in AWS Marketplace, you can pull the container images of Scalar products from the private container registry ([ECR](https://aws.amazon.com/ecr/)) of AWS Marketplace. This section explains how to deploy Scalar products on your [EKS](https://aws.amazon.com/eks/) cluster from the private container registry.
+If you subscribe to Scalar products in AWS Marketplace, you can pull the container images of Scalar products from the private container registry ([ECR](https://aws.amazon.com/ecr/)) of AWS Marketplace. This section explains how to deploy BYOL Scalar products on your [EKS](https://aws.amazon.com/eks/) cluster from the private container registry.
```
Comment (josh-wong) [inline docs/AwsMarketplaceGuide.md:115]:
```suggestion
By subscribing to Scalar products in the AWS Marketplace, you can pull the container images of Scalar products from the private container registry ([ECR](https://aws.amazon.com/ecr/)) of the AWS Marketplace. This section explains how to deploy Scalar products with the BYOL option in your [EKS](https://aws.amazon.com/eks/) cluster from the private container registry.
```

File: docs/AwsMarketplaceGuide.md [hunk 5]
```diff
@@ -1,15 +1,17 @@
 # How to install Scalar products through AWS Marketplace
 
-Scalar products (Scalar DB, Scalar DL, and their tools) are provided in AWS Marketplace as Container Image. This guide explains how to install Scalar products through AWS Marketplace.
+Scalar products (ScalarDB, ScalarDL, and their tools) are provided in AWS Marketplace as Container Image. This guide explains how to install Scalar products through AWS Marketplace.
 
-Note that some Scalar products are licensed under commercial licenses, and the AWS Marketplace provides them as BYOL (Bring Your Own License). Please make sure you have appropriate licenses.
+Note that some Scalar products are licensed under commercial licenses, and the AWS Marketplace provides them as Pay-As-You-Go and BYOL (Bring Your Own License). If you select Pay-As-You-Go, AWS charges you our product license fee. If you select BYOL, please make sure you have appropriate licenses.
```
[the remark this answers is not in the document]
Comment (kota2and3kan) [reply docs/AwsMarketplaceGuide.md:5]: Ah, yes. I think it is better to say `based on your usage`. I will update this sentence based on your suggestion! Thank you!

**Comment by kota2and3kan** [comment]
@josh-wong  
Thank you for your review!
I applied your suggestions.
Please take a look again when you have time!

**Review by josh-wong — APPROVED** [review]
LGTM👍 Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**